ginivision/src/main/java/com/ortiz/touch/TouchImageView.java

```diff
@@ -89,7 +88,7 @@
     //
     // Size of view and previous view size (ie before rotation)
     //
-    private int viewWidth, viewHeight, prevViewWidth, prevViewHeight;
+    private int prevViewWidth, prevViewHeight;

```

Comment:
Had to alter the third-party `TouchImageView` to make it work with resize animations.